lib/tapioca/dsl/compilers/url_helpers.rb

```diff
@@ -102,6 +102,9 @@ class << self
           def gather_constants
             return [] unless defined?(Rails.application) && Rails.application
 
+            routes_reloader = Rails.application.routes_reloader
```

Comment:
Might be worth adding a comment here linking to https://github.com/rails/rails/pull/51614